<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_variablenwerte_anzeigen</class>
 <widget class="QDialog" name="Dialog_variablenwerte_anzeigen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_bez">
          <property name="text">
           <string>Bezeichnung:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_zeilennummer">
          <property name="text">
           <string>Zeilennummer:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_bez"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_zeilennummer"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Aktualisieren</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_wert_beschriftungstext">
        <property name="text">
         <string>Wert:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_wert">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
